superset-frontend/src/welcome/DashboardTable.tsx

```diff
@@ -58,6 +68,13 @@ class DashboardTable extends React.PureComponent {
         row: {
           original: { url, dashboard_title: dashboardTitle },
         },
+      }: {
+        row: {
+          original: {
+            url: string;
+            dashboard_title: string;
+          };
+        };
```

Comment:
Sorry for missing this earlier, but I think all `Cell` renderers actually have the same signature. 

`react-table` has typing for its parameters, so this should probably be refactored as:

```ts
import { Column } from 'react-table'

interface DataboardRecord {
  url: string;
  dashboard_title: string;
  changed_on: string;
  changed_by_name: string;
  changedByUrl: string;
}

columns: Column<DataboardRecord>[] = [{
   ...
   Cell: ({ row: { original: { ... } } }) => ...
}]
```